lib/VM/JSNativeFunctions.cpp

```diff
@@ -122,6 +91,8 @@ static llvm::DenseMap<const void *, const char *> funcNames() {
 #undef NATIVE_CONSTRUCTOR
 #undef NATIVE_CONSTRUCTOR_TYPED
   };
+
+  int numNames = sizeof names / sizeof *names;
```

Comment:
This doesn't seem right. `names` is just an array of char. Probably `functionPointers` instead?

Also, using `int` causes a warning in the assert() below, breaking the build with `-Werror`.

Reply:
Thanks for tracking down my dumb mistake